components/nofrendo/nes/nes.h

```diff
@@ -36,7 +36,7 @@
 
 /* Visible (NTSC) screen height */
 #ifndef NES_VISIBLE_HEIGHT
-#define  NES_VISIBLE_HEIGHT   224
+#define  NES_VISIBLE_HEIGHT   240
```

Comment:
Why did you change this?

Reply:
This here causes the emulator to break due to memory allocation.